playbooks/tasks/certificate_authority.yml

```diff
@@ -30,6 +30,9 @@
     ansible_become: "{{ ansible_become_localhost }}"
 
 - name: Apt update
+  tags:
+    - certificate_authority
```

Comment:
this task is not related to certificate. Package tag looks okay

Reply:
the tasks that have certificate_authority tag in them are mostly using openssl. So if those tasks need to be run separately using --tags certificate_authority then running this task previous to them would ensure when they are run they dont give the error command openssl not found.